common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3230,6 +3236,18 @@
     ],
     "sqlState" : "0A000"
   },
+  "STATE_STORE_MULTIPLE_VALUES_PER_KEY" : {
+    "message" : [
+      "Store does not support multiple values per key"
+    ],
+    "sqlState" : "42802"
+  },
+  "STATE_STORE_UNSUPPORTED_OPERATION" : {
+    "message" : [
+      "<operationType> operation not supported with <entity>"
+    ],
+    "sqlState" : "XXKST"
```

Comment:
Can a user get here? If so that would imply a 0A*** sqlstate rather than an internal error.

Reply:
No, a user cannot get here. This can stay as an internal error.